src/Camelotia.Presentation/ViewModels/ProviderViewModel.cs

```diff
@@ -92,12 +94,32 @@ public sealed class ProviderViewModel : ReactiveObject, IProviderViewModel, IAct
                 () => Path.GetDirectoryName(CurrentPath), 
                 canCurrentPathGoBack);
 
-            _open.Merge(_back)
+            _setPath = ReactiveCommand.Create<string, string>(path => path);
+
+            _open.Merge(_back).Merge(_setPath)
```

Comment:
Nit: formatting. Colt is working on a Roslyn Analyzer that could autoformat such stuff btw https://twitter.com/GameConstructor/status/1290802364497051648
```suggestion
            _open.Merge(_back)
                .Merge(_setPath)
```